templates/lxc-download.in

```diff
@@ -54,9 +53,13 @@ LXC_NAME=
 LXC_PATH=
 LXC_ROOTFS=
 
-# Deal with GPG over http proxy
-if [ -n "${http_proxy:-}" ]; then
-    DOWNLOAD_KEYSERVER="hkp://p80.pool.sks-keyservers.net:80"
+if [ -z "${DOWNLOAD_KEYSERVER+x}" ]; then
```

Comment:
That should be: "${DOWNLOAD_KEYSERVER:-}" to match the code style used in the rest of the file.